app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,58 @@
+class OrganizationsController < ApplicationController
+  before_action :set_organization, only: [:show, :edit, :update, :destroy]
+
+  # GET /organizations
+  def index
+    @organizations = Organization.all
+  end
+
+  # GET /organizations/1
+  def show
+  end
+
+  # GET /organizations/new
+  def new
+    @organization = Organization.new
+  end
+
+  # GET /organizations/1/edit
+  def edit
+  end
+
+  # POST /organizations
+  def create
+    @organization = Organization.new(organization_params)
+
+    if @organization.save
+      redirect_to @organization, notice: 'Organization was successfully created.'
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.